src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,199 @@
+    1014047902397333505 | canceled | {defaultdb.public.office_dogs} | 2024-10-24 17:30:25+00        | 2024-10-21 17:54:20.797643+00 | LWW                      | LOGICAL REPLICATION STREAM into defaultdb.public.office_dogs from external://cluster_a
+    ~~~
+
+1. Drop the LDR job on both clusters. Canceling the LDR streams will remove the history retention job, which will cause the data to be garbage collected according to the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) setting. Use [`CANCEL JOB`]({% link {{ page.version.version }}/cancel-job.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CANCEL JOB {ldr_job_id};
+    ~~~
+
+1. Perform the schema change on cluster A. 
+1. Drop the existing table from cluster B. 
+1. Recreate the table and its new schema on cluster B. 
+1. Create new LDR streams for the table on both clusters A and B. Run `CREATE LOGICAL REPLICATION STREAM` from the **destination** cluster for each stream:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+    ~~~
+
+#### Coordinate schema changes for unidirectional LDR 
+
+If you have a unidirectional LDR setup, you should cancel the running LDR stream and redirect all application traffic to the source cluster.
+
+1. Drop the LDR job on the **destination** cluster. Canceling the LDR job will remove the history retention job, which will cause the data to be garbage collected according to the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) setting. Use [`CANCEL JOB`]({% link {{ page.version.version }}/cancel-job.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CANCEL JOB {ldr_job_id};
+    ~~~
+
+1. Once the job has `canceled`, perform the required schema change on both the source and destination tables.
+1. Start a new LDR job from the **destination** cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+    ~~~
+
+## Jobs and LDR
+
+You can run any other CockroachDB [job]({% link {{ page.version.version }}/show-jobs.md %}) on any cluster that is involved in an LDR job. Both source and destination clusters in LDR are active, which means they can both serve production reads and writes as well as run [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}), [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}), and so on. 
```

Comment:
"can run any other job" is not quite right, for example, you cannot run import on an LDR job right now. 

i would only discuss backup and changefeed here.

Reply:
Specified cdc/ backup here